react/package.json

```diff
@@ -77,6 +77,7 @@
         "deck.gl": "^8.8.11",
         "deep-equal": "^2.0.5",
         "fast-json-patch": "^3.0.0-1",
+        "gl-matrix": "^3.4.3",
```

Comment:
Should also push `package-lock.json` so that we get this package when we run `npm ci`